Entities/Common/FireScripts/IsFlammable.as

```diff
@@ -81,6 +81,9 @@ void onTick(CBlob@ this)
 	{
 		server_setFireOff(this);
 		this.set_netid("burn starter blob", 0);
+		
+		CSprite@ sprite = this.getSprite();
+		sprite.PlaySound("/ExtinguishFire.ogg");
```

Comment:
Add a `null` check for the sprite just in case it doesn't exist for some reason.

Reply:
Done.